src/pages/DokumentOversikt/components/statuspanel/Statuspanel.tsx

```diff
@@ -0,0 +1,49 @@
+import './statuspanel.less';
+
+import React from 'react';
+import { Normaltekst } from 'nav-frontend-typografi';
+
+interface CircleProps {
+    number: number;
+    checked?: boolean;
+}
+
+const Circle = ({ number, checked }: CircleProps) => {
+    return (
+        <div className={`circle ${checked ? 'circle--checked' : ''}`}>
+            <p>{checked ? '✓' : number}</p>
+        </div>
+    );
+};
+
+interface Statuselement extends CircleProps {
+    topptekst: string;
+    bunntekst: string;
+}
+
+export interface StatuspanelProps {
+    statusElementer: Statuselement[];
+}
+
+const Statuspanel = ({ statusElementer }: StatuspanelProps) => {
+    return (
+        <div className="full-width">
+            <div className="limit statuslinje">
+                {statusElementer.map(statuselement => {
```

Comment:
Vet ikke om dette er personlig preferanse, men jeg foretrekker at slike mappings som dette genereres i en egen variabel utenfor JSX.

Det jeg ville gjort:

1. Lag en ny const StatusElement som lager elementet beskrevet i linje 35 -> 41 under
2. På linje 29, før return: const statusElementer = statusElementerData.map(data => <StatusElement ... />);

Variabelnavn er bare for eksempel, finn på noe mer passende 🤔